netty/src/test/java/io/grpc/netty/NettyClientHandlerTest.java

```diff
@@ -457,6 +458,141 @@ public void ping_failsWhenChannelCloses() throws Exception {
   }
 
   @Test
+  public void dataPingSentOnDataRecieved() throws Exception{
+    createStream();
+    ByteBuf frame = dataFrame(3, false); 
+    channelRead(frame);
+    assertEquals(1, handler().pingcount);
```

Comment:
Please add a new line between channelRead and assertEquals

In general, tests should follow the three clause structure: SETUP, TEST, VERIFY.  Each section should be separated by a new line.  This makes it easy to see what is being tested.